src/orderbook.js

```diff
@@ -164,7 +201,15 @@ OrderBook.EVENTS = [
   'offer_changed', 'offer_funds_changed'
 ];
 
-OrderBook.DEFAULT_TRANSFER_RATE = 1000000000;
+OrderBook.DEFAULT_TRANSFER_RATE = Amount.from_json(1000000000);
+
+OrderBook.NOTIFY_THROTTLE_TIME = 4000;
```

Comment:
In my firefox it takes 300-600 ms for `autobridgeCalculator.calculate` to complete, and 
`computeAutobridgedOffersWrapper` can take 400-1500 ms
With you suggested change, compute can be called every 250 ms, so it will definitely will kill users cpu in case of high load.
What we can do is debounce throttled function - that way we will get benefit of both methods.